parm/ccpp_regional_c768_FA_HRRTMG.nml.IN

```diff
@@ -0,0 +1,308 @@
+&amip_interp_nml
+       interp_oi_sst = .true.
+       use_ncep_sst = .true.
+       use_ncep_ice = .false.
+       no_anom_sst = .false.
+       data_set = 'reynolds_oi',
+       date_out_of_range = 'climo',
+/
+
+ &atmos_model_nml
+       blocksize = 32
+       chksum_debug = .false.
+       dycore_only = .false.
+       fdiag = 1
+       ccpp_suite = '@[CCPP_SUITE]'
+/
+
+&diag_manager_nml
+       prepend_date = .F.
+/
+
+ &fms_io_nml
+       checksum_required   = .false.
+       max_files_r = 100,
+       max_files_w = 100,
+/
+
+ &fms_nml
+       clock_grain = 'ROUTINE',
+       domains_stack_size = 3000000,
+       print_memory_usage = .false.
+/
+
+ &fv_grid_nml
+       grid_file = 'INPUT/grid_spec.nc'
+/
+
+ &fv_core_nml
+       layout   = 24,20
```

Comment:
This will need to be changed to 20,24 to avoid non-uniform block sizes (we should also check all other regional_c768 tests that we set up recently).